scheduler/services/resource/ocs_resource_service.py

```diff
@@ -90,7 +91,7 @@ class OcsResourceService(ResourceManager, metaclass=Singleton):
         'PinholeC': 'PinholeC'
     }})
 
-    def __init__(self, sites: FrozenSet[Site] = ALL_SITES):
+    def __init__(self, sites: FrozenSet[Site] = ALL_SITES, with_post_init: bool = True):
```

Comment:
Not quite clear what's going on with the `_load_post_init`. Can you clarify and maybe add some comments? I'm getting a bit lost in this PR.

Reply:
As FileOCSResourceService and OCSResourceService shared __init__ if the loading of the files are inside there as they were the File service would do double duty on loading information it won't be used. So that flag helps to keep the OCSResourceService behavior and to not load the files in the FileResourceService.